src/pineko/cli/compare.py

```diff
@@ -9,14 +9,14 @@
 
 
 @command.command("compare")
-@click.argument("pineappl_path", type=click.Path(exists=True))
 @click.argument("fktable_path", type=click.Path())
+@click.argument("grid_path", type=click.Path(exists=True))
 @click.argument("max_as", type=int)
 @click.argument("max_al", type=int)
-@click.argument("pdf", type=str)
+@click.argument("pdfs", type=click.STRING, nargs=2)
```

Comment:
this broke the benchmarks and I'm not sure it was a good patch, because now you _have to_ pass two PDFs

Reply:
I guess the only option is then `-1` with a check?

Reply:
[4aac0f8](https://github.com/NNPDF/pineko/pull/181/commits/4aac0f89842abac08497017ad87fb8e4fd0e987b)